src/NuGet.Clients/NuGetConsole.Host.PowerShell/RegistryHelper.cs

```diff
@@ -24,11 +24,9 @@ public static bool CheckIfPowerShell2OrAboveInstalled()
                 return true;
             }
 
-            // PS 3.0 and 4.0 is set under "...\PowerShell\3" key
+            // PS 3.0 and higher is set under "...\PowerShell\3" key
             keyValue = GetSubKeyValue(@"SOFTWARE\Microsoft\PowerShell\3\PowerShellEngine", "PowerShellVersion");
-            if ("3.0".Equals(keyValue, StringComparison.OrdinalIgnoreCase)
-                ||
-                "4.0".Equals(keyValue, StringComparison.OrdinalIgnoreCase))
+            if(double.TryParse(keyValue, out double result) && result >= 3.0)
```

Comment:
This can be problematic on machines with culture different than US. In some countries comma is used as a separator. Among other alternatives are: using `SemVer` parser (preferrable), specify neutral culture, provide custom formatter.